### PR TITLE
Parameterize fixes

### LINKED ISSUE #84: V-93147,9 Check/Fix text need to be parameterized
V-93147,9 Check/Fix text need to be parameterized  to allow overlays to display alternative warning/legal banners in these fields.

### LINKED ISSUE #85: V-93509 parameterize \"300\" in check/fix text
Check and Fix text baseline has \"300\" MaxConnIdleTime specified, but this needs to be replaced with the input "maximum_idle_time" to allow overlays to display correct wording.

### PR DESCRIPTION
**Pull request by karikarshivani**
fixes #84 
fixes #85 

Signed-off-by: karikarshivani <karikarshivani@gmail.com>

### CODE REVIEW

**Comment by karikarshivani** [comment]
Update on V-93149: All the elements in the input array `LegalNoticeCaption` will now be shown in the check and fix text.
[Report Reference](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/files/9600094/V-93149Array.json.txt)

**Review by ejaronne — APPROVED** [review]
Looks good. Thanks!

**Comment by ejaronne** [comment]
V-93479 text needs to be parameterized using the password_history_size input

**Comment by ejaronne** [comment]
https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/blob/parameterize/controls/V-93463.rb needs to have its title, check, and fix text updated to reflect the test results based on input('minimum_password_length')

**Comment by ejaronne** [comment]
Parameterized V-93209 https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/a9500345c47d16cc57ea67b7f722df15e93c8ff3

**Comment by ejaronne** [comment]
Parameterized V-93285 based on #{input('maximum_password_age_machine')}
https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/1c04a91a48b242c8cc9b65127244885c9d1e46b6

**Comment by karikarshivani** [comment]
> https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/blob/parameterize/controls/V-93463.rb needs to have its title, check, and fix text updated to reflect the test results based on input('minimum_password_length')

Done! https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/c24567cb3da1f029b159e3621e6d830ee87b673b

File: controls/V-93285.rb [hunk 1]
```diff
@@ -14,9 +14,9 @@
     Value Name: MaximumPasswordAge
 
     Value Type: REG_DWORD
-    Value: 0x0000001e (30) (or less, but not 0)"
-  desc  "fix", "This is the default configuration for this setting (30 days).
-    Configure the policy value for Computer Configuration >> Windows Settings >> Security Settings >> Local Policies >> Security Options >> \"Domain member: Maximum machine account password age\" to \"30\" or less (excluding \"0\", which is unacceptable)."
+    Value: 0x0000001e (#{input('maximum_password_age_machine')}) (or less, but not 0)"
```
Comment (karikarshivani) [inline controls/V-93285.rb:17]: > Parameterized V-93285 based on #{input('maximum_password_age_machine')} [1c04a91](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/commit/1c04a91a48b242c8cc9b65127244885c9d1e46b6)

Parameterized some values that were still 30 https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/86437f3f6d97a370d96ce8cfc573eeecb07b3f1f
However, the hex value `0x0000001e` in the check text might be tricky to parameterize. I think we have a few options:
1. Remove the hex value altogether
2. Hardcode the hex value to `0x0000003c` (60) in the overlay as that is the only possible value for ARS HVA
3. Use Ruby to automatically convert integers (like 30 or 60) to hex by using: `30.to_s(16)`

If we use option 3, the code would look something like this:
```
Value Type: REG_DWORD
Value: 0x000000#{input('maximum_password_age_machine').to_s(16)} (#{input('maximum_password_age_machine')}) (or less, but not 0)
```

Comment (ejaronne) [reply controls/V-93285.rb:17]: Let's go ahead and try option 3. Thanks!

Comment (karikarshivani) [reply controls/V-93285.rb]: Implemented option 3 https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/86/commits/9acc401409129867cc0fffb0531dd30c90febfce

**Review by ejaronne — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**